.builds/freebsd.yml

```diff
@@ -12,19 +12,22 @@ packages:
 sources:
   - https://github.com/alacritty/alacritty
 
+environment:
+  PATH: /home/build/.cargo/bin:/usr/bin/
```

Comment:
Would you mind adding `/usr/local/bin` before `/usr/bin`, since it's where things goes from packges on  freebsd iirc.